src/sentry/static/sentry/app/views/organizationGroupDetails/reprocessingDialogForm.tsx

```diff
@@ -30,18 +32,27 @@ const impacts = [
   ),
 ];
 
+const remainingEventsChoices: [string, string][] = [
```

Comment:
you don't need to add the return type here, because the compiler will automatically detect the type